doc/content/guides/getting-started/upgrading.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Upgrading"
+description: ""
```

Comment:
I think we do need a weight here for order. The getting started is one of those where it makes sense